src/test/java/org/gridsuite/filter/expertfilter/NumberExpertRuleTest.java

```diff
@@ -1475,100 +1481,100 @@ private static Stream<Arguments> provideArgumentsForVoltageLevelTest() {
         Mockito.when(voltageLevel1.getHighVoltageLimit()).thenReturn(Double.NaN);
 
         return Stream.of(
-            // --- EQUALS --- //
```

Comment:
Revert the tab plz

Reply:
DONE in https://github.com/gridsuite/filter/pull/17/commits/a38bac0d0331005ddbbac3a6981e982f97ee0005